### PR TITLE
Biometrics cache

### PR DESCRIPTION
**Pull request by elainamcgovern**
# Description

Modified to store the history of entries beyond just the most recent one. 

Fixes # 72

## Bug Fix or Feature

- [ ] Bug
- [ ] Feature
- [ ] Other

# Testing Instructions

- [ ] Start services after cleaning up containers
- [ ] insert user1 into the user table through docs
- [ ] populate location and biometrics information
- [ ] start Unity project
- [ ] see that events are being logged 

## Linting

- [ ] I've linted my code

### CODE REVIEW

File: ARSIS-Unity/Assets/ARSIS/Tools/Caches/BiometricsCache.cs [hunk 1]
```diff
@@ -7,9 +7,13 @@ public class BiometricsCache : MonoBehaviour
 {
 
     public BiometricsEvent BiometricsEvent;
+    private static int MINUTES = 5;
+    private static int MAX_ENTRYS = 60*MINUTES;
+    private List<BiometricsEvent> BiometricsList;eee
     public static BiometricsCache BiometricsCacheSingleton { get; private set; }
```
Comment (isaacdenny) [inline ARSIS-Unity/Assets/ARSIS/Tools/Caches/BiometricsCache.cs:13]: Looks like you have a typo here: "eee" after the semicolon

**Review by isaacdenny — CHANGES_REQUESTED** [review]
Everything looks good except one part (see comments)

File: ARSIS-Unity/Assets/ARSIS/Tools/Caches/BiometricsCache.cs (hunk 1 above)
Comment (elainamcgovern) [reply ARSIS-Unity/Assets/ARSIS/Tools/Caches/BiometricsCache.cs:13]: I think this should be fixed now, let me know if it's not! :)

**Review by isaacdenny — APPROVED** [review]
Looks good! great job... sorry for the delayed review

**Pull request merged**